nuitka/utils/CStrings.py

```diff
@@ -62,19 +64,18 @@ def encodePythonUnicodeToC(value):
     """Encode a string, so that it gives a wide C string literal."""
     assert type(value) is unicode, type(value)
 
-    result = ""
+    result = []
 
     for c in value:
         cv = ord(c)
-
-        if c == "\\":
-            result += "\\\\"
-        elif 34 < cv < 128:
-            result += c
+        if cv == 92:  # ord('\\')
+            result.append(r"\\")
+        elif 34 < cv < 128:  # between '!' and '~'
+            result.append(c)
         else:
-            result += r"\x%04x" % cv
+            result.append(r"\x%04x" % cv)
 
-    return 'L"%s"' % result
+    return 'L"' + "".join(result) + '"'
```

Comment:
I forgot what we last figured out, but I wonder, if the change should not go the other direction, like the `34` < `128` part could be done with strings, there is a way to encode them

```
>>> chr(128)
'\x80'
>>> chr(34)
'"'
```

The the ord value would only be needed in the else branch, which is supposed to be rare.

Reply:
With last figure out, I seem to recall that in-place string operations are not slower at all. I forgot if that held up or not though.